### PR TITLE
Connectors wave 1

### PR DESCRIPTION
**Pull request by mesellings**
## Description

This PR contains the first wave of the Connector docs enhancements (https://github.com/camunda/developer-experience/issues/356).

Preview: https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/introduction-to-connectors/

Changes can be summarised as follows:

- Content is now structured into 3 areas, so clearer and quicker for users to find content.
- Added a popular/new section to the landing page.
- Added a banner at the bottom of main landing pages to cross-link to the marketplace.
- On the [main Connectors page](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/out-of-the-box-connectors/available-connectors-overview/) layout changed to match the home page cards, and clickable badges added for connector type.
- Added connector logos and used marketplace/product ones where possible, but some don't lend themselves well to our content (tiny text) so some are different.
- Removed the "connector" suffix from the left-hand menu items.
- Main connector landing pages had an initial overhaul (ahead of wave 3).
- Started converting "Connector" to lower-case on landing pages as part of the Connector casing work.

Reviewers: This work basically gives us a restructured menu and improves the main connector landing pages.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [X] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@sbuettner Although still a WIP, I managed to spend a few days this week on the connectors wave 1 work (I've had to bring some of the landing page work in as well). 

You can preview this at:

https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/introduction-to-connectors/

* You can see the content is now structured into 3 areas, should be much clearer and quicker for users to find content.
* I added a popular/new section to the landing page - wdyt?
* Added a banner to cross-link to the marketplace.
* On the [main Connectors page](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/out-of-the-box-connectors/available-connectors-overview/) I changed the layout to match the home page tiles, and used clickable badges instead of a dropdown for a better UX.
* I've added connector logos and used marketplace/product ones where possible, but some don't lend themselves well to our content (tiny text) so some are different. I can work with @YanaSegal on these.
* I removed the "connector" suffix from the left-hand menu items, much clearer now.
* I still have to look at the custom connectors landing page, that would be my next job.
* This work gives us a restructured menu etc, and improves the main landing pages, which would leave the template and individual pages to tackle in separate PRs for waves 2 and 3 (@christinaausley and I will tackle this).
* I have started making "Connector" lower-case as part of the Connector casing work.

I wouldn't do a review yet until it is ready, but I just wanted to update you on some good progress - we should be able to get this out in January sometime. 🤞 

FYI: @akeller @crobbins215

**Comment by YanaSegal** [comment]
@mesellings this looks great! i really like the new structure! super useful, thank you!
let me know how can I help :) 
looking forward to wave 2/3.

**Comment by sbuettner** [comment]
@mesellings Everyone who looked at this was at awe and I havent heard about anything worth correcting.

Thank you so much and lets get this merged!

Things that we have to watch closely from my perspective:

**Comment by mesellings** [comment]
@sbuettner This PR for the first wave of connectors work is now ready for review and merge 👍 

@pepopowitz as requested, I've tagged you in for a review of the react component work on [Camunda connectors](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/out-of-the-box-connectors/available-connectors-overview/)

Reviews should only concentrate on the restructure and main pages, as most other pages are untouched and will be covered in waves 2 and 3:
- [Connectors](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/introduction-to-connectors/)
- [Camunda connectors](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/out-of-the-box-connectors/available-connectors-overview/)
- [Custom connectors](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/custom-built-connectors/build-connector/)

**Note:** I haven't changed any file locations, so no redirects are required.

**Review by conceptualshark — COMMENTED** [review]
This PR is basically an inspiration at this point -- amazing work getting this all together! I find it incredibly easy to navigate with this structure, and I love the cleanup of the titles as a quick win while you're in here. The connectors table especially is a nice upgrade. 

I am sure others will have reviews more relevant to their areas of expertise, but from an organizational and basic information perspective, no notes from me!

**Review by akeller — COMMENTED** [review]
First off, this looks world class and I'm so thankful for you @mesellings and bringing this amazing experience to the Connectors team. They truly deserve it. 🧡 

Comment: I thought the "Find a Camunda Connector" button would take me to the Camunda Marketplace, but I see that's another button.  

Similar to my brand police comment on your dark mode PR, please ensure that you are following brand guidelines for the additional colors of labels and buttons (outbound and inbound labels prompted my brain to write this). **This could be blocking.** 

Question: For the latest/popular content, who will commit to maintaining/updating that moving forward? I usually shy away from highlights like this because they can go stale. 

Question: Did you mean to have a bolded link here? 
image

Based on the other page, I assumed this would be a button.

Question: Is the > in the "Find a Camunda Connector >" button intentional? When do you add it and when do you not (the Marketplace button doesn't have a >, but pops you out of the docs).

**Comment by mesellings** [comment]
Thanks @akeller, great review points, addressed as follows:

> Comment: I thought the "Find a Camunda Connector" button would take me to the Camunda Marketplace, but I see that's another button.

Not sure how else we could reword this as a button - I feel this should be the main CTA on the page so users can straight away find docs for a connector, possibly "Find docs for a Camunda connector"?

> Similar to my brand police comment on your dark mode PR, please ensure that you are following brand guidelines for the additional colors of labels and buttons (outbound and inbound labels prompted my brain to write this). **This could be blocking.**

I've asked the brand team in #ask-marketing for advice and to check this pattern, and have asked @karl-heinsen for advice as well - it's tricky as I'm not sure we have a pattern for this in Carbon currently. The colours are on-brand though, so compliant in that respect.
> 
> Question: For the latest/popular content, who will commit to maintaining/updating that moving forward? I usually shy away from highlights like this because they can go stale.

This is a question for @sbuettner and my main concern also with this - I originally added these in as a protoype, but feel they are a good addition. In terms of maintenance, it is easy to add/edit these items, and I can show the team how to do this. Latest could just be updated per alpha or major release, and Popular can be updated on any cadence we choose - ad-hoc, monthly, release, when we want to highlight a particular connector for example. @sbuettner do you feel the team could manage this, or do you want these taken out instead?
> 
> Question: Did you mean to have a bolded link here? image
> Based on the other page, I assumed this would be a button.

I did as we don't have a pattern in docs for this kind of link - I don't want a secondary link on the page like this to be a button - buttons should be very sparing, e.g. the two buttons currently. However, I've rejigged this so we can kick this can further down the road for now!

> Question: Is the > in the "Find a Camunda Connector >" button intentional? When do you add it and when do you not (the Marketplace button doesn't have a >, but pops you out of the docs).

It was, and follows a pattern in the corporate site, but for consistency I removed this until we can look properly at the docs site styles - keeping it vanilla!

Thanks, all great comments, hopefully this resolves them (once label colours are brand approved) 👍

**Comment by sbuettner** [comment]
Regarding:  latest/popular content

> @sbuettner do you feel the team could manage this, or do you want these taken out instead?

I like it but without proper maintenance if can quickly get out-of-date and thats something we want to avoid. Maybe we can build something more automated based on the actual marketplace but for now I would remove all things that require additional maintenance for the team.

**Comment by mesellings** [comment]
> Regarding: latest/popular content
> 
> > @sbuettner do you feel the team could manage this, or do you want these taken out instead?
> 
> I like it but without proper maintenance it can quickly get out-of-date and that is something we want to avoid. Maybe we can build something more automated based on the actual marketplace but for now I would remove all things that require additional maintenance by the team.

I think it would be a shame to remove this, and it would be good to see from analytics over six months or so how much traction/usage this sections gets? I would be happy to maintain this on a per-release basis, or another option would be to change it just featured/popular - this allows us to highlight the connectors we want to push/people come to the docs for?

**Comment by YanaSegal** [comment]
I completely agree with @mesellings 
>I think it would be a shame to remove this...
, and I believe it would bring great value to our users. 
How big is the maintenance effort? This can be part of the publishing process. Similar to how we must submit to the Marketplace. And maybe some of it can be automated.

**Comment by mesellings** [comment]
> > I think it would be a shame to remove this...
> 
> I completely agree with @mesellings , and I believe it would bring great value to our users. How big is the maintenance effort? This can be part of the publishing process. Similar to how we must submit to the Marketplace. And maybe some of it can be automated.

It's probably about an hour work in total, so not much - just some text edits and new logos if they don't already exist - as I say, I can handle doing this as part of release work until we have a better idea of how much people use/value it?

**Comment by mesellings** [comment]
> > I think it would be a shame to remove this...
> 
> I completely agree with @mesellings , and I believe it would bring great value to our users. How big is the maintenance effort? This can be part of the publishing process. Similar to how we must submit to the Marketplace. And maybe some of it can be automated.

Maintenance is small and not complicated - an hour or so each time?

**Comment by sbuettner** [comment]
> I would be happy to maintain this on a per-release basis until this time, and then decide there?

@mesellings That would work for me. If you agree then lets keep it as is. I dont see the need to push anyone to use a Connector hence we can keep latest/popular.

**Comment by mesellings** [comment]
FYI @akeller I have changed all the active filter colours to be the Camunda blue, and kept the label colours the same currently, as per suggestion by @karl-heinsen in a Slack convo - this matches the same pattern as our primary interactive button color (blue) when in the active state. 👍

**Review by YanaSegal — APPROVED** [review]
great value to our users!!!

**Comment by mesellings** [comment]
We've had the nod of approval from branding (https://camunda.slack.com/archives/C01UMP0V17Z/p1737366091530889), so I think this PR is all good to go - @sbuettner are you happy for us to release this?

**Comment by sbuettner** [comment]
@mesellings Yes, lets go! 🚀 

Thanks again for all the effort to make Connectors looking this great.

**Comment by mesellings** [comment]
@pepopowitz did you want to review the react component on [Camunda connectors](https://preview.docs.camunda.cloud/pr-4794/docs/next/components/connectors/out-of-the-box-connectors/available-connectors-overview/)?

File: docs/components/connectors/img/connector-amazon-sns.png
Comment (pepopowitz) [inline docs/components/connectors/img/connector-amazon-sns.png:1]: non-blocking: Some of these images are rather large compared to the size we emit them on the page. It is probably worth resizing them using something like [imagemagick](https://usage.imagemagick.org/resize/#resize), to improve load time of them.

File: docs/components/connectors/introduction.md [hunk 1]
```diff
@@ -1,41 +1,183 @@
...
+    <img src={IconConnectorTypesImg} alt="Connector types"/>
+    <h3>Connector types</h3>
+    <p>Learn about <a href="../use-connectors/inbound" title="Inbound Connectors">Inbound</a> and <a href="../use-connectors/outbound" title="Outbound Connectors">Outbound</a> connector types.</p>
+  </a>
+    <a href="../out-of-the-box-connectors/available-connectors-overview" class="connector-card" title="Get started with Connectors">
+    <img src={IconConnectorImg} alt="How to use Connectors"/>
+    <h3>Camunda connectors</h3>
+    <p>Find technical documentation for prebuilt Camunda connectors.</p>
+  </a>
+</div>
```
Comment (pepopowitz) [inline docs/components/connectors/introduction.md:64]: potentially blocking: I noticed that this bit is not formatted (e.g. extra indentation on multiple lines). I would have expected prettier to take care of this for you during commit. 

The fact that it didn't suggests to me that this doc is not well-formed. It may be no big deal if that's the case, but I think it's worth investigating what is causing this to not format.

File: docs/components/connectors/introduction.md [hunk 2]
```diff
@@ -1,41 +1,183 @@
...
+  <a href="../custom-built-connectors/connector-templates" class="connector-card" title="Connector templates">
+    <img src={IconConnectorTemplateImg} alt="Connector templates icon" class="connector-card-image"/>
+    <h3>Connector templates</h3>
+    <p>Create, generate, and manage connector templates.</p>
+  </a>
+    <a href="../custom-built-connectors/connector-sdk" class="connector-card" title="Connector SDK">
+    <img src={IconConnectorSdkImg} alt="Connector SDK icon" class="connector-card-image"/>
+    <h3>Connector SDK</h3>
+    <p>Use the SDK to create your own custom Java connectors.</p>
+  </a>
```
Comment (pepopowitz) [inline docs/components/connectors/introduction.md:89]: non-blocking but encouraged: For the amount of repetition we do with these, it probably makes sense to extract a reusable component for all of these. It would potentially save us time for modifying the cards in the future, and would turn these call sites into something like:

```react
  <ConnectorCard href="../custom-built-connectors/connector-sdk" 
                 title="Connector SDK"
                 icon={IconConnectorSdkImg}
                 description="Use the SDK to create your own custom Java connectors."
  />
```

File: docs/components/connectors/introduction.md [hunk 3]
```diff
@@ -1,41 +1,183 @@
...
+    <p>Connect your BPMN service to an email POP3, IMAP or SMTP server.</p>
+  </a>
+</div>
+
+<div class="connector-small-grid">
+  <a href="../out-of-the-box-connectors/email/?email=inbound" class="connector-small-link">
+    <div class="connector-small">
+      <img src={EmailImg} alt="Email inbound connector"/>Email inbound
+    </div>
+  </a>
```
Comment (pepopowitz) [inline docs/components/connectors/introduction.md:126]: See my previous comment re: a reusable component for cards. If we do extract a component for those cards, we'll want one for these small links too.

File: docs/components/connectors/react-components/_banner-marketplace.md
Comment (pepopowitz) [inline docs/components/connectors/react-components/_banner-marketplace.md:1]: 👍 I appreciate and support this abstraction for reuse.

File: docs/components/connectors/react-components/_connectors-table.css
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.css:1]: non-blocking: do you plan to handle accounting for dark mode of these components in the dark mode PR, or in this PR in preparation? I noticed some colors in here that I assume handle light mode only.

File: docs/components/connectors/react-components/_connectors-table.js [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+  const [filterType, setFilterType] = useState("All");
+
+  const connectors = [
+    {
+      name: "Amazon Bedrock",
+      description: "Interact with Amazon Bedrock from your processes.",
+      type: "Outbound",
+      link: "../amazon-bedrock",
+      image: AmazonBedrockImg,
+    },
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.js:53]: non-blocking but encouraged: this array would work nicely as an extracted data structure that's imported into here, and into the introduction page, to avoid duplication of these images, descriptions, links, etc. From the introduction page, since you want specific items in specific places, you could access them from the source structure by name. 

In terms of future maintenance, I think it's always helpful to have one source of the data. Multiple usually leads to  drift, when someone updates in one place but not the other.

File: docs/components/connectors/react-components/_connectors-table.css [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+}
+
+.badge--active--outbound {
+  background-color: #007bff;
+  color: #fff;
+}
+.badge--active--inbound {
+  background-color: #007bff;
+  color: #fff;
+}
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.css:280]: non-blocking question: I couldn't identify why the inbound and outbound tabs need a dedicated active state. Does `.badge--active` not cover them? If not, removing these 8 lines and adjusting the associated toggles in the React component would reduce complexity a tiny bit.

File: docs/components/connectors/react-components/_connectors-table.js [hunk 6]
```diff
@@ -0,0 +1,527 @@
...
+        <span
+          className={`badge--default ${
+            filterType === "Outbound" ? "badge--active--outbound" : ""
+          }`}
+          onClick={() => handleFilterType("Outbound")}
+        >
+          Outbound
+        </span>
+      </div>
+      <span style={{ fontSize: "14px" }}>
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.js:470]: non-blocking: Any particular reason for inline styling here but CSS classes elsewhere?

File: docs/components/connectors/react-components/_connectors-table.js [hunk 7]
```diff
@@ -0,0 +1,527 @@
...
+
+  return (
+    <div className="connector-input">
+      <input
+        type="text"
+        placeholder="Find a Camunda connector"
+        value={searchTerm}
+        onChange={handleSearch}
+        className="connector-input-box"
+        style={{ marginRight: "20px" }}
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.js:442]: non-blocking: can this inline style be pushed into the associated CSS class?

File: docs/components/connectors/react-components/_connectors-table.js [hunk 8]
```diff
@@ -0,0 +1,527 @@
...
+                <p>{connector.description}</p>
+              </div>
+              <div className="connector-type">
+                <span
+                  className={`badge ${
+                    connector.type === "Inbound"
+                      ? "badge--beginner"
+                      : connector.type === "Outbound"
+                      ? "badge--medium"
+                      : "badge--default"
```
[the remark this answers is not in the document]
Comment (pepopowitz) [reply docs/components/connectors/react-components/_connectors-table.js:493]: blocking: This indentation suggestion illustrates that nested ternaries are usually confusing to read 😅 

At the very least, I recommend parentheses grouping the individual ternaries to alleviate this confusion. 

Better might be to define `.badge--inbound` and `.badge--outbound` classes that have more semantic meaning than the ones we're currently tapping into, so that this code can be reduced to something along the lines of ``className={`badge $badge--{connector.type.toLower()}`} `` (no promises on the exact API for string.toLower but I know it's something like that)

**Review by pepopowitz — CHANGES_REQUESTED** [review]
This is fantastic and really impressive!!!

I have one blocking comment (the nested ternary) and a handful of other suggestions, mostly for improved maintainability.

File: docs/components/connectors/react-components/_connectors-table.css [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+  margin-right: 10px;
+  font-size: 85%;
+  font-weight: var(--ifm-font-weight-bold);
+  line-height: 1;
+  color: #777;
+}
+
+.badge--default:hover {
+  background-color: #ccc;
+}
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.css:266]: I forgot to mention in my review that the contrast doesn't meet accessibility recommendations on the `default` and `hover` states of these buttons (the two gray buttons in this screenshot):

image

My browser tools recommended `#3a3a3a` as the lightest shade of gray that contrasts with both backgrounds enough to meet AAA/optimal compliance.

File: docs/components/connectors/react-components/_connectors-table.css (hunk 5 above)
Comment (mesellings) [reply docs/components/connectors/react-components/_connectors-table.css:280]: We did have different colours, but only just decided with branding to use the single blue colour, so I just changed them to that but I think it's worth keeping them like this for now in case we do change things again, just to avoid rework, at least until it is 'definite' that this is the pattern we'll use - good spot though 👍

File: docs/components/connectors/react-components/_connectors-table.js (hunk 7 above)
Comment (mesellings) [reply docs/components/connectors/react-components/_connectors-table.js:442]: Yup, done 👍

File: docs/components/connectors/react-components/_connectors-table.js (hunk 4 above)
Comment (mesellings) [reply docs/components/connectors/react-components/_connectors-table.js:53]: Agreed, but perhaps something to look at once the connectors pages are launched and settled - we may remove the cards from the intro page if they get no traction/too annoying to keep updated, so I'll leave this for now, and we can see after six months? I don't want to hold the connectors work up for this if that's okay?

File: docs/components/connectors/introduction.md (hunk 3 above)
Comment (mesellings) [reply docs/components/connectors/introduction.md:126]: See previous comment - one for the future

File: docs/components/connectors/img/connector-amazon-sns.png
Comment (mesellings) [reply docs/components/connectors/img/connector-amazon-sns.png:1]: Good idea, they are all still pretty small in file size - the plan is to use them in individual connector pages, so I kept them at this size to avoid having two images for each connector (small/large). But, we can look at this suggestion later, and if we don't end up using them elsewhere, we can always scale the images down?

File: docs/components/connectors/introduction.md (hunk 1 above)
Comment (mesellings) [reply docs/components/connectors/introduction.md]: This is weird - do you have a recommendation for how we should format HTML in markdown?

Comment (pepopowitz) [reply docs/components/connectors/introduction.md]: My recommendation is for prettier to do it for us 😅 Usually, it does. The fact that it didn't hear suggests that something in this document is not well-formed HTML, that's usually what prevents prettier from auto-formatting it for us. 

What I would do with it is remove code until prettier formatted it (checking with "format document" in vs code for a tight feedback loop). If that doesn't uncover something specific, then I guess leave it unformatted, or do your best to indent things appropriately. (Siblings are typically indented at the same level, which isn't the case for lines 50, 54, and 59. Also, extracting a component for these would reduce the amount of adjustment to do here.)

File: docs/components/connectors/introduction.md (hunk 2 above)
Comment (pepopowitz) [reply docs/components/connectors/introduction.md:89]: > ideally we can swizzle the main card component to be more like these 

These seem uniquely different from the basic card, and I do not think they are the same abstraction. This sounds like a separate issue and separate work.

> decided to develop my own card component for this work

What I'm suggesting is you build your own _reusable_ card component, rather than copy-pasting the entire markup for every single item. 

I'm not sure if I'm communicating clearly. I agree that you should be using your own card component. I don't think that what you have here is your own card component, it's repeated markup. This is a non blocking suggestion, but I'm not sure my feedback is landing.

File: docs/components/connectors/introduction.md [hunk 10]
```diff
@@ -46,21 +46,21 @@ Connectors are often configured as a [BPMN process](/components/concepts/process
...
+<a href="../connector-types" class="connector-card" title="Connector types">
+<img src={IconConnectorTypesImg} alt="Connector types"/>
+<h3>Connector types</h3>
+<p>Learn about <a href="../use-connectors/inbound" title="Inbound Connectors">Inbound</a> and <a href="../use-connectors/outbound" title="Outbound Connectors">Outbound</a> connector types.</p>
+</a>
+<a href="../out-of-the-box-connectors/available-connectors-overview" class="connector-card" title="Get started with Connectors">
+<img src={IconConnectorImg} alt="How to use Connectors"/>
+<h3>Camunda connectors</h3>
+<p>Find technical documentation for prebuilt Camunda connectors.</p>
+</a>
```
Comment (pepopowitz) [inline docs/components/connectors/introduction.md:63]: Removing all indentation is not an improvement over what was there; I think we should revert this commit!

**Review by pepopowitz — APPROVED** [review]

**Comment by mesellings** [comment]
@pepopowitz I've created react components for the two card types on the introduction as suggested (I wanted to leave this work until later, but figured it was worth doing now to get it out of the way) - can you please check what I've done is okay? (this also solves the HTML formatting issue) - thank you so much! 🚀

File: docs/components/connectors/introduction.md (hunk 2 above)
Comment (mesellings) [reply docs/components/connectors/introduction.md]: See latest commit for the (actual) card components 👍

File: docs/components/connectors/react-components/_connectors-card-sml.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+          alt={`${title} icon`}
+          className="connector-small-image"
+        />
+        {title}
+      </div>
+    </a>
+  );
+};
+
+const ConnectorsGridSml = ({ connectors }) => {
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-card-sml.js:19]: non-blocking: renaming this `ConnectorsGridSmall` and the file `connectors-card-small` would add clarity (I had to take some time to think what sml stood for 😅).

File: docs/components/connectors/react-components/_connectors-table.js [hunk 12]
```diff
@@ -0,0 +1,519 @@
...
+                <p>{connector.description}</p>
+              </div>
+              <div className="connector-type">
+                <span
+                  className={`badge badge--${connector.type.toLowerCase()}`}
+                >
+                  {connector.type}
+                </span>
+              </div>
+            </a>
```
Comment (pepopowitz) [inline docs/components/connectors/react-components/_connectors-table.js:492]: non-blocking: visually, the only difference I can see between the cards rendered on the introduction page vs the grid results is the presence of a badge. I would have expected to see the extracted Card component able to conditionally render a badge if it's provided, so that the card itself is only defined in one place.

**Review by pepopowitz — APPROVED** [review]
The extraction of these components aligns with what I was thinking! 

I do think there are some more improvements that can be made for the maintainability of these components. I suspect it is going to be you maintaining these React components, in which case I defer to you to decide on making the improvements.

**Comment by mesellings** [comment]
Final change(!) - having merged in dark mode to the docs, I've added some pre-emptive styling and image edits to make sure this new connectors content is optimised for dark mode as well.

@akeller @sbuettner @YanaSegal Can I please get a final approval so I can merge this work please? Thank you 🙏

File: docs/components/connectors/img/Untitled-1.png
Comment (akeller) [inline docs/components/connectors/img/Untitled-1.png:1]: This needs a more descriptive name.

Comment (akeller) [reply docs/components/connectors/img/Untitled-1.png:1]: Actually, is this used anymore?

**Review by akeller — APPROVED** [review]
Approving, but please address the image comment in a future PR ASAP.

File: docs/components/connectors/img/Untitled-1.png
Comment (mesellings) [reply docs/components/connectors/img/Untitled-1.png:1]: Thanks, good spot! No, this isn't used anymore, I must have left it in when mocking up the banner - I'll remove it in a follow-up PR 👍
